src/metabase/queries/metadata.clj

```diff
@@ -1,19 +1,45 @@
 (ns metabase.queries.metadata
   (:require
+   [metabase.api.common :as api]
```

Comment:
There was a conflict here, it was missing the `[clojure.set :as set]` `:require` from master, and we should just leave it out.

https://github.com/metabase/metabase/blob/d848a88e645b56c16b1631861312cd437a6a1d7e/src/metabase/queries/metadata.clj#L2-L4